eclipse-theia/theia
# fix-9205: Fill task options explicitly if problemMatchers set

#### What it does

Fixes issue [9205](https://github.com/eclipse-theia/theia/issues/9205). Was originally proposed in #9207 by @vnfedotov. This one resolves conflicts with the stale changes.

#### How to test


Please see https://github.com/eclipse-theia/theia/pull/9207#issuecomment-800549676

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
